README.md

```diff
@@ -493,16 +495,77 @@ If it is not possible to use existing nf-core modules, you can create your own m
 
 ### 5.2.1. Module software requirements
 
-Each module should define its software requirements as a docker/singularity container. This requires using the `container` keyword in the process. For example:
+Each module should define its software requirements as a singularity/docker container. This requires using the `container` keyword in the process. For example:
 
 ```
 container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
     'https://depot.galaxyproject.org/singularity/fastqc:0.11.9--0' :
     'biocontainers/fastqc:0.11.9--0' }"
 ```
 
+Since IRIDA Next Nextflow pipelines aim to follow [nf-core](https://nf-co.re/docs/usage/getting_started/configuration#docker-registries) standards, the default container registry should be set to the publicly available `quay.io` registry in the `nextflow.config` file using the `.registry` parameters. For example:
+
+```
+singularity.registry =  'quay.io'
+docker.registry =       'quay.io'
+```
+
 For more information, see the [Nextflow containers][] documentation and the [nf-core modules software requirements][] guide.
 
+### 5.2.2 Configuring Module Software with Private or Alternate Container Registries
+
+To configure a private or alternate container registry, the standard practice is to set the `.registry` option to point to the desired resistry in the `nextflow.config` file. For example:
+
+```
+docker.registry = 'container-registry.com'
+```
+
+This setting automatically prefixes each container name with the specified registry URL. In the above example, `biocontainers/fastqc:0.11.9--0` is transformed into `container-registry.com/biocontainers/fastqc:0.11.9--0`, ensuring that the container is downloaded from `container-registry.com`.
+
+In IRIDA Next Nextflow pipelines, we have chosen to adhere to nf-core standards by designating `quay.io` as the default container registry. This means that, by default, containers such as `biocontainers/fastqc:0.11.9--0` are transformed into `quay.io/biocontainers/fastqc:0.11.9--0`. This configuration can be limiting if the pipeline requires containers from multiple registries (such as both `quay.io` and `docker.io`) since only one default registry can be configured at a time. Allowing flexibility in sourcing containers from different registries is useful in scenarios where certain containers are only available in specific registries or where there are preferences or restrictions on registry use.
+
+To address this, we use `process.ext.override_configured_docker_registry` to manage the selection of docker container registries dynamically by adding the following line to the `nextflow.config` file:
```

Comment:
I was thinking, in theory this override could be applied to all supported container types, so should we change it to `process.ext.override_configured_container_registry`?